src/auth/Logout.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { userSignedOut } from 'src/auth/auth-events/signout';
+import * as Nav from 'src/libs/nav';
+
+export const logoutCallbackLinkName = 'logout-callback';
+export const Logout = () => {
+  try {
+    userSignedOut();
+  } catch (e) {
+    console.error(e);
+  }
+  Nav.goToPath('root');
+  return <div />;
```

Comment:
Side effects should go in effect hooks. Also, since we're only interested in the side effect, this doesn't need to render an element.

```suggestion
export const Logout = (): ReactNode => {
  useEffect(() => {
    try {
      userSignedOut();
    } catch (e) {
      console.error(e);
    }
    Nav.goToPath('root');
  }, []);
  return null;
```